scenario-databases.rst

```diff
@@ -75,18 +75,20 @@ Scenario processing
 -------------------
 
 When submitting a scenario (a.k.a. "run") to an IIASA database instance, the server
-executes a scenario-processing workflow including *region-aggregation* and
-*scenario validation* prior to saving the scenario to the database. The processing uses
-the **nomenclature** package (`read the docs <https://nomenclature-iamc.readthedocs.io>`_).
+executes a scenario-processing workflow including *region-aggregation* and *scenario
+validation* prior to saving the scenario to the database. The processing uses the
+**nomenclature** package (`read the docs <https://nomenclature-iamc.readthedocs.io>`_).
 
 The region-aggregation and validation is configured via a project-specific GitHub_
 repository, usually named `https://github.com/iiasa/<project>-workflow`_. Please contact
 the respective project managers or the Scenario Services team if you need access.
 
-You can also run the project workflow locally (on your computer) before submission to
-an IIASA database instance, to make sure that the validation and processing works.
-See :ref:`local-processing` for more information!
+The workflow for processing files uploaded via the IIASA Scenario Explorer is
```

Comment:
I'd keep the paragraph on local-processing (including the link) and add the new paragraph below.